_site/docs/execution/observability.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: Observability
+parent: Execution
+nav_order: 7
+---
+
+# Observability
+
+Executions are organized by a hierarchy of ownership, and indexed with client-specified fields. The [longrunning.operations](https://cloud.google.com/service-infrastructure/docs/service-management/reference/rpc/google.longrunning) API can be used to find a configuration-limited number of executions organized in this hierarchy by `name` and `filter`.
+
+## Hierarchy
+
+ * Correlated Invocations - specified by build tool invoker
+   * Tool Invocation - generated internally by build tool
+     * Execution - generated internally by buildfarm
+
+An *Execution* is an individual Execute request made from a client to buildfarm. The names for all Executions on buildfarm will be the string `<instance name>/executions/<uuid>`, with `uuid` being generated after the Execute call is made.
+
+A *Tool Invocation* is an organization of all requests made by a client during one logical user invocation. For instance, `bazel`, as a _tool_, will generate a UUID per _invocation_, usually by a user or CI script - e.g: each time you run `bazel`, it generates internally and uses a different `toolInvocationId`, making this a collection of _Executions_ (and other requests) per run.
+
+A set of *Correlated Invocations* is uniquely identified by another UUID, the `correlatedInvocationsId`, which is expected to encompass one or more _Tool Invocations_. A build tool will provide the mechanism to specify this by a user - e.g: each time you run `bazel`, you can specify the same `--build_request_id` to indicate that a run is a member of a collection of _Tool Invocations_. The correlatedInvocationsId must be a string that _ends_ with an 8-4-4-4-12 format UUID string (important because this value is completely specified by a client user).
```

Comment:
consider documenting what happens if it _doesn't_ end in 8-4-4-4-12 UUID. Is it ignored? not correlated? Buildfarm crashes? 😂